app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -23,7 +23,7 @@ def perform(form526_submission_id)
         with_tracking(self.class.name, form526_submission.saved_claim_id, form526_submission_id) do
           return if form526_submission.pending_eps?
 
-          client = Lighthouse::VeteransHealth::Client.new(get_icn(form526_submission))
+          client = Lighthouse::VeteransHealth::Client.new(form526_submission.get_icn)
```

Comment:
Note that `app/workers/rapid_ready_for_decision/disability_compensation_job.rb` is only used as a backup Sidekiq job: https://github.com/department-of-veterans-affairs/vets-api/blob/01e46b8bdf62e8bb940e93459446297cfa017082/app/services/rapid_ready_for_decision/constants.rb#L12

Let's not modify it.